recursio-client/UI/Menus/GameRoomLobby.gd

```diff
@@ -7,21 +7,50 @@ export var icon_not_ready: Texture
 signal btn_leave_pressed()
 
 onready var _game_room_name: Label = get_node("Content/TopBar/GameRoomName")
-onready var _player_list: ItemList = get_node("Content/PlayerList")
+onready var _player_list: ItemList = get_node("Content/HBoxContainer/PlayerList")
 
 onready var _btn_ready: Button = get_node("Content/BottomBar/Btn_Ready")
 onready var _btn_leave: Button = get_node("Content/BottomBar/Btn_Leave")
 
+
+onready var _level_names = [
+	"H",
+	"|- -|"
+]
+
+onready var _level_preview_icons = [
```

Comment:
would be better to move those to the definition of the levels and mapping them to scenes (below in StartMenu)